### PR TITLE
Backup upload images projects

### LINKED ISSUE #259: Import export all projects via the web interface
A reflexion for the problem for migrate from IOU VM to GNS3 VM.

Perhaps in the web interface we can add an options for sending a zip with all the content of the project and images directory as zip files and another options for reuploading projects and all images.

### PR DESCRIPTION
**Pull request by noplay**
Fix #259

### CODE REVIEW

**Comment by noplay** [comment]
Due to the fact it's touch the master I prefer to have a validation from someone else before merging.

File: gns3server/handlers/upload_handler.py [hunk 1]
```diff
@@ -59,29 +62,98 @@ def upload(request, response):
...
+                    destination_dir = os.path.join(UploadHandler.image_directory(), data["type"])
+                    destination_path = os.path.join(destination_dir, data["file"].filename)
+                    os.makedirs(destination_dir, exist_ok=True)
+                    with open(destination_path, "wb+") as f:
+                        chunk = data["file"].file.read()
+                        f.write(chunk)
+                    st = os.stat(destination_path)
+                    os.chmod(destination_path, st.st_mode | stat.S_IXUSR)
         except OSError as e:
+            print(e)
```
Comment (grossmj) [inline gns3server/handlers/upload_handler.py]: Do you want to keep this print?

File: gns3server/handlers/upload_handler.py [hunk 2]
```diff
@@ -59,29 +62,98 @@ def upload(request, response):
...
+
     @staticmethod
     def image_directory():
         server_config = Config.instance().get_section_config("Server")
         return os.path.expanduser(server_config.get("images_path", "~/GNS3/images"))
+
+    @staticmethod
+    def project_directory():
+        server_config = Config.instance().get_section_config("Server")
+        return os.path.expanduser(server_config.get("projects_path", "~/GNS3/images"))
```
Comment (grossmj) [inline gns3server/handlers/upload_handler.py]: Should that be ~/GNS3/projects?

File: gns3server/handlers/upload_handler.py (hunk 1 above)
Comment (noplay) [inline gns3server/handlers/upload_handler.py]: :+1:

File: gns3server/handlers/upload_handler.py (hunk 2 above)
Comment (noplay) [inline gns3server/handlers/upload_handler.py]: :+1:

**Pull request merged**